org.freedesktop.Piper.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "org.freedesktop.Piper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "piper",
+      "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.ratbag1"
+      ],
+    "modules": [
+        {
+            "name": "python-evdev",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-deps ."
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/da/c9/8f88a77b6f934d60e3ff0e210ed7a15f47a2e9c05e4c8fea67d99d670ab1/evdev-1.0.0.tar.gz",
+                    "sha256": "be0288ef1125bf1c539bb8f3079ef4aa5fb813af28f0c5294a4e744ee554398a"
+                }
+            ]
+        },
+        {
+            "name": "python-lxml",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-deps ."
+            ],
+            "build-options": {
+                "env": {
+                    "XSLT_CONFIG": "pkg-config libxslt"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/ca/63/139b710671c1655aed3b20c1e6776118c62e9f9311152f4c6031e12a0554/lxml-4.2.4.tar.gz",
+                    "sha256": "e2afbe403090f5893e254958d02875e0732975e73c4c0cdd33c1f009a61963ca"
+                }
+            ]
+        },
+        {
+            "name": "Piper",
+            "buildsystem": "meson",
+            "builddir" : true,
```

Comment:
Can remove this, it is forced for Meson: https://github.com/flathub/flathub/pull/601#discussion_r214521469